deploy/charts/operator-crds/templates/toolhive.stacklok.dev_toolconfigs.yaml

```diff
@@ -0,0 +1,113 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.17.3
+  name: toolconfigs.toolhive.stacklok.dev
+spec:
+  group: toolhive.stacklok.dev
+  names:
+    kind: ToolConfig
+    listKind: ToolConfigList
+    plural: toolconfigs
+    shortNames:
+    - tc
+    singular: toolconfig
```

Comment:
I believe it should be plural, but I'm wondering, do we want to add an `MCP` prefix to this, as in `MCPToolsConfig`?

I don't know what the best practice is when writing new CRDs.